extensions/wikia/WikiFactory/tests/WikiFactoryLoaderIntegrationTest.php

```diff
@@ -8,12 +8,17 @@ class WikiFactoryLoaderIntegrationTest extends WikiaDatabaseTest {
 	/** @var string $dbName */
 	private $dbName;
 
+	/** @var Language $oldContentLanguage */
```

Comment:
Can add the explanation you provided in the PR as a comment here, for context?

Reply:
Added